README.md

```diff
@@ -13,8 +13,10 @@ cd plotIt/
 
 # Initialize the git remotes
 source firstsetup.sh 
-# Load the proper environment (a la CMSSW_7_4_10)
-source setup_sl6_env.sh
```

Comment:
If you remove that, you have to source `setup_for_cms_env.sh` otherwise it won't work